plugins/source/aws/resources/services/cloudformation/stack_instances.go

```diff
@@ -62,9 +63,13 @@ func fetchStackInstanceSummary(ctx context.Context, meta schema.ClientMeta, pare
 		if err != nil {
 			return err
 		}
-		for i := range page.Summaries {
+		for _, summary := range page.Summaries {
+			if summary.StackId == nil {
```

Comment:
It seems like we could have some duplicate PKs if StackId is null? Can it happen that a single stack set has multiple stacks in an error state? I don't see a better way of handling this but maybe it's something to document

Reply:
IDK if there can be multiple stack instances in a bad state (inoperable?) inside a single stack set.
I can add a more comprehensive comment to this, but IDK if it's a good idea to add this to a doc somewhere.

Reply:
Why is it not a good idea to mention this in the table description? If it can happen, it's a limitation users should be aware of without needing to read the code

Reply:
cf41bfebb1156798d9d923c7654c0145bcbf0aeb